backend/src/index.ts

```diff
@@ -234,9 +234,9 @@ class Server {
     if (config.STATISTICS.ENABLED && config.DATABASE.ENABLED) {
       statisticsRoutes.initRoutes(this.app);
     }
-    if (Common.indexingEnabled()) {
-      miningRoutes.initRoutes(this.app);
-    }
+    // if (Common.indexingEnabled()) {
```

Comment:
why not just remove it instead of commenting it

Reply:
removed and added comment